frontend/src/lib/api.ts

```diff
@@ -1852,6 +1853,23 @@ const api = {
         }
         return response
     },
+
+    async loadPaginatedResults(
```

Comment:
This function existed in 2 places, I was about to put it in a 3rd, so I consolidated here.